src/main/java/com/heimtn/telegrambot/entity/NotificationTask.java

```diff
@@ -0,0 +1,80 @@
+package com.heimtn.telegrambot.entity;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * To the created table via liquibase, make a class with Entity annotation.
```

Comment:
no, it's not because of liquibase. It needed to Hibernate work correctly as well as Spring Repositories